### DISCUSSION TITLE
How to nominate an additional 3rd-party reference source? Any data flow diagrams?

### DISCUSSION THREAD

**Discussion by jasnow** [cloudsecurityalliance/gsd-tools#168] [Q&A]
* How to nominate an additional 3rd-party reference source? 

    * I appears the project has two types of references. I saw 3 mentioned somewhere and "3rd party references" mentioned inside some adversaries.

* Have any data flow (DAG) diagrams (upstream, downstream)?

Thanks

**Comment by kurtseifried** [comment]
We can have as many reference as needed, the more the merrier. As for upstream/downstream that data could be for example included in SBOM data that we would happily allow in namespaces from those organizations.
